virttest/utils_net.py

```diff
@@ -4351,3 +4351,50 @@ def delete_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     return utils_misc.cmd_status_output(tmux_cmd, shell=True, verbose=True,
                                         ignore_status=ignore_status,
                                         session=session)
+
+
+def create_linux_bridge(linux_bridge_name, iface_name, ignore_status=False):
+    """
+    Create linux bridge and connect a physical interface to the bridge via tmux command on local host
+    1. use tmux to avoid the network broken;
+    2. add a physical interface to the bridge, and set the bridge up;
+
+    :param linux_bridge_name: The linux bridge
+    :param iface_name: the physical interface to be attached to the bridge
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: bridge created or raise exception
+    """
+    # Create bridge
+    br_path = "/sys/class/net/%s" % linux_bridge_name
+    if not utils_package.package_install(['tmux', 'dhcp-client']):
+        raise exceptions.TestError("Failed to install the required packages.")
+    if os.path.exists(br_path):
+        raise exceptions.TestError("Create bridge error as there is already an interface named '%s' on the host"
+                                   % linux_bridge_name)
+    cmd = 'tmux -c "ip link add name {0} type bridge; ip link set {1} up;' \
+          ' ip link set {1} master {0}; ip link set {0} up;' \
+          ' pkill dhclient; sleep 6; dhclient {0}; ifconfig {1} 0"'.format(linux_bridge_name, iface_name)
+    s, o = utils_misc.cmd_status_output(cmd, shell=True, verbose=True, ignore_status=ignore_status)
```

Comment:
since o is not used, better change to s,_ = utils_misc()

Reply:
utils_misc.cmd_status_output will handle the exception and return status and stdout. So I think the previous code is correct. It should return utils_misc.cmd_status_output. How do you think? @yalzhang @chunfuwen

Reply:
I think chunfu means  "s,_ = utils_misc.cmd_status_output"  instead of "s,o=utils_misc.cmd_status_output". So I think it is ok to use "_" to replace "o"